amr-wind/turbulence/LES/AMD.H

```diff
@@ -45,179 +53,85 @@ private:
-                            cm1 * vel(i - 1, j, k, 1)) *
-                           idx;
-    const amrex::Real wx = (cp1 * vel(i + 1, j, k, 2) + c * vel(i, j, k, 2) +
-                            cm1 * vel(i - 1, j, k, 2)) *
-                           idx;
-    const amrex::Real tx = (cp1 * theta(i + 1, j, k) + c * theta(i, j, k) +
-                            cm1 * theta(i - 1, j, k)) *
-                           idx;
-
-    cp1 = Stencil::c10;
-    c = Stencil::c11;
-    cm1 = Stencil::c12;
-
-    const amrex::Real uy = (cp1 * vel(i, j + 1, k, 0) + c * vel(i, j, k, 0) +
-                            cm1 * vel(i, j - 1, k, 0)) *
-                           idy;
-    const amrex::Real vy = (cp1 * vel(i, j + 1, k, 1) + c * vel(i, j, k, 1) +
-                            cm1 * vel(i, j - 1, k, 1)) *
-                           idy;
-    const amrex::Real wy = (cp1 * vel(i, j + 1, k, 2) + c * vel(i, j, k, 2) +
-                            cm1 * vel(i, j - 1, k, 2)) *
-                           idy;
-    const amrex::Real ty = (cp1 * theta(i, j + 1, k) + c * theta(i, j, k) +
-                            cm1 * theta(i, j - 1, k)) *
-                           idy;
-
-    cp1 = Stencil::c20;
-    c = Stencil::c21;
-    cm1 = Stencil::c22;
-
-    const amrex::Real uz = (cp1 * vel(i, j, k + 1, 0) + c * vel(i, j, k, 0) +
-                            cm1 * vel(i, j, k - 1, 0)) *
-                           idz;
-    const amrex::Real vz = (cp1 * vel(i, j, k + 1, 1) + c * vel(i, j, k, 1) +
-                            cm1 * vel(i, j, k - 1, 1)) *
-                           idz;
-    const amrex::Real wz = (cp1 * vel(i, j, k + 1, 2) + c * vel(i, j, k, 2) +
-                            cm1 * vel(i, j, k - 1, 2)) *
-                           idz;
-    const amrex::Real tz = (cp1 * theta(i, j, k + 1) + c * theta(i, j, k) +
-                            cm1 * theta(i, j, k - 1)) *
-                           idz;
-
-    const amrex::Real dx2 = dx * dx;
-    const amrex::Real dy2 = dy * dy;
-    const amrex::Real dz2 = dz * dz;
-    const amrex::Real num_shear =
-        -2.0 * C *
-        (ux * (ux * ux * dx2 + uy * uy * dy2 + uz * uz * dz2) +
-         vy * (vx * vx * dx2 + vy * vy * dy2 + vz * vz * dz2) +
-         wz * (wx * wx * dx2 + wy * wy * dy2 + wz * wz * dz2) +
-         (uy + vx) * (ux * vx * dx2 + uy * vy * dy2 + uz * vz * dz2) +
-         (uz + wx) * (ux * wx * dx2 + uy * wy * dy2 + uz * wz * dz2) +
-         (vz + wy) * (vx * wx * dx2 + vy * wy * dy2 + vz * wz * dz2));
-    amrex::Real num_buoy =
-        C * beta * (dx2 * wx * tx + dy2 * wy * ty + dz2 * wz * tz);
-    amrex::Real denom = (ux * ux + uy * uy + uz * uz + vx * vx + vy * vy +
-                         vz * vz + wx * wx + wy * wy + wz * wz) +
-                        1e-15;
+    amrex::Real num_shear = 0;
+    amrex::Real num_buoy = 0;
+    amrex::Real denom = 0;
+    const int n_ind = idxOp(i, j, k);
+    amrex::Real h = nlo + (n_ind + 0.5) * dx[normal_dir];
```

Comment:
I think there's a bunch you can `const` here: `h`, `gradTbar_h`, `diuj`, `sij`, `dkui` etc ?